src/System.Runtime/tests/System/Enum.cs

```diff
@@ -1,78 +1,82 @@
-// Licensed to the .NET Foundation under one or more agreements.
+﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
 using System;
-using System.Runtime.Tests.Common;
-
 using Xunit;
 
 public static class EnumTests
 {
     [Theory]
-    [InlineData("Red", false, true, SimpleEnum.Red)]
-    [InlineData(" Red", false, true, SimpleEnum.Red)]
-    [InlineData("Red ", false, true, SimpleEnum.Red)]
-    [InlineData(" red ", true, true, SimpleEnum.Red)]
-    [InlineData(" Red , Blue ", false, true, SimpleEnum.Red | SimpleEnum.Blue)]
-    [InlineData("Blue,Red,Green", false, true, SimpleEnum.Red | SimpleEnum.Blue | SimpleEnum.Green)]
-    [InlineData("Blue,Red,Red,Red,Green", false, true, SimpleEnum.Red | SimpleEnum.Blue | SimpleEnum.Green)]
-    [InlineData("Red,Blue,   Green", false, true, SimpleEnum.Red | SimpleEnum.Blue | SimpleEnum.Green)]
-    [InlineData("B", false, true, SimpleEnum.B)]
-    [InlineData("B,B", false, true, SimpleEnum.B)]
```

Comment:
Why did you remove this?  It was explicitly being used to test 1-char-long enum names.